models/edr/dbt_artifacts/dbt_exposures.sql

```diff
@@ -2,7 +2,10 @@
   config(
     materialized='incremental',
     transient=False,
-    post_hook='{{ elementary.upload_dbt_exposures() }}'
+    post_hook='{{ elementary.upload_dbt_exposures() }}',
+    unique_key='unique_id',
+    on_schema_change='append_new_columns',
```

Comment:
Let's talk with Itamamr about this? maybe we should use here the sync columns mode